test/Dapr.E2E.Test/ServiceInvocation/InvokeMethodTest.cs

```diff
@@ -0,0 +1,30 @@
+namespace Dapr.E2E.Test
+{
+    using System;
+    using System.Net.Http.Json;
+    using System.Text.Json;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Dapr.Client;
+    using Xunit;
+
+    public class Main
+    {
+        [Fact]
+        public async void TestServiceInvocation()
+        {
+            var testApp = new DaprApp("testapp", 5000);
+            try
+            {
+                var (daprHttpEndpoint, _) = testApp.Start();
+                var client = DaprClient.CreateInvokeHttpClient(appId: "testapp", daprEndpoint: daprHttpEndpoint);
```

Comment:
Should there be a dapr side process for the test as well? This is going to use a single sidecar, but in the product scenarios there would be two right? How does Java approach this?

Reply:
I thought the tests should focus on how sdk interacts with the dapr side car. Java tests use this same approach.

Reply:
If this is the approach that java uses I think that's fine. I dont' want to reinvent the wheel.